src/Nop.Plugin.Payments.SwedbankPayCheckout/Nop.Plugin.Payments.SwedbankPayCheckout.csproj

```
@@ -0,0 +1,54 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Copyright>COPYRIGHT Swedbank Pay</Copyright>
+    <Company>Swedbank Pay</Company>
+    <Authors>Swedbank Pay and contributors</Authors>
+    <PackageLicenseUrl>..\..\LICENSE</PackageLicenseUrl>
```

I think the `PackageLicenseUrl` should be an absolute and full URL, not a relative local path. Although it doesn't exist since the `LICENSE` file is isn't merged to `master` yet, I think we can set the URL to what it's going to be once this PR is merged.

```suggestion
    <PackageLicenseUrl>https://github.com/SwedbankPay/swedbank-pay-nopcommerce/blob/master/LICENSE</PackageLicenseUrl>
```